build.gradle

```diff
@@ -3,7 +3,7 @@ task wrapper(type: Wrapper) {
 }
 
 allprojects {
-    version = "2.0.6"
+    version = "2.0.7"
```

Comment:
What happens if we fix a bug in the no-player library and there is no exo-player update, which version name are we going to have if we align it?